CHANGELOG.md

```diff
@@ -7,27 +7,133 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.5.5] - 03-12-20
+### Added
+
+- Added `Walker` trait, to specify the way in which a `Branch` or `BranchMut` can be walked from a root node
+- Added `Nth` trait, to construct a branch to the nth element of a `Compound` collection
+- Added `MaxKey` and `Keyed` traits to keep track of the maximum keyed leaf in the collection
+- Added the `levels` method to `Branch` to introspect the individual branch `Levels`
+- Added the `path` constructor to `Branch` and `BranchMut` to traverse the collectiong along a specified path
+- Added the `MappedBranch` to provide branches that only allow access to certain parts of its leaves
+- Added implementation of `IntoIterator` for `Branch` and `BranchMut`
+- Added `First` auto-trait to construct a `Branch` to the first element in a collection
+- Added `LinkedList` implementation in tests
+
+### Changed
+
+- Changed `canonical`/`canonical_derive` version from 0.5 to 0.6
+- Split up the `Annotation` trait into `Annotation` for the leaves, and `Combine` for the nodes.
+- Changed the iterator on `Compound` to only iterate over populated subtrees or leaves.
```

Comment:
few nits:

1. The section is past tense ("Added", "Changed", etc) but the item list inside are present tense ("Add", "Change", etc)
2. if we can't rephrase the "Split up" into "Change", I would suggest to use "Refactor" that is already an alias for "Change"
3. Do not use the period at the end of the list item.

Reply:
got it